Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/Main.js

```diff
@@ -222,8 +222,21 @@ function documentReady()
         row.appendChild(cell);
 
         cell = document.createElement("td");
-        var view = new BuildbotBuilderQueueView(platformQueues.builders);
-        cell.appendChild(view.element);
+        var queues = {};
+        var generators = [];
+        platformQueues.builders.forEach(function(queue) {
+            var generator = queue.viewGenerator ? queue.viewGenerator : BuildbotBuilderQueueView;
```

Comment:
@aproskuryakov asked why we have a default of `BuildbotBuilderQueueView` instead of setting the `viewGenerator` appropriately. And the answer is that `BuildbotQueue` is the underlying `Queue` object for `BuildbotBuilderQueueView` and a few other `Buildbot*QueueView` objects.